scripts/ciTest.js

```diff
@@ -3,6 +3,15 @@ var cp = require("child_process");
 var path = require("path");
 var fs = require("fs");
 
+var duneBinDir = path.join(
```

Comment:
nit: several `.js` scripts define this

Reply:
Did not want to put it into bin_path.js as it is only needed when building the compiler.
Not sure if it's worth creating a separate file for this.

Reply:
Ok, created one now. 🙂